resources-ktx/build.gradle.kts

```diff
@@ -1,19 +1,17 @@
-import com.redmadrobot.build.dependencies.androidx
-import com.redmadrobot.build.dependencies.jetbrains
-
 plugins {
-    id("com.redmadrobot.android-library")
-    id("com.redmadrobot.publish")
+    convention.library.android
     id("kotlin-parcelize")
 }
 
 version = "1.3.1-0"
 description = "A set of Kotlin extensions for accessing resources"
 
+android {
+    namespace = "com.redmadrobot.extensions.resources-ktx"
+}
+
 dependencies {
-    api(jetbrains.kotlin.stdlib)
-    api(androidx.annotation)
     api(androidx.fragment)
-    implementation(androidx.core)
-    implementation(androidx.appcompat.resources)
+    api(androidx.core)
+    api(androidx.appcompat.resources)
```

Comment:
Oh. Was it me who said to make these dependencies `api`? They should be `implementation` as they are not a part of the library's public API

Reply:
that's mine 🙀